.github/workflow_scripts/test_tabular.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -ex
+
+source $(dirname "$0")/env_setup.sh
+
+setup_build_env
+setup_mxnet_gpu
```

Comment:
Why are we installing MXNet for Tabular tests? Tabular never uses MXNet, and I don't think it uses MXNet in Jenkins either.